tasks/users.yml

```diff
@@ -5,13 +5,13 @@
     name: "{{ postgresql_service_name }}"
     state: started
 
-- name: PostgreSQL | Make sure the PostgreSQL users are present
+- name: "PostgreSQL | Make sure the PostgreSQL users are present"
```

Comment:
Nowhere else in the role are double-quotes used, so please remove them to maintain the style